src/sentry/static/sentry/app/components/list/symbol/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import Numeric from './numeric';
+import Bullet from './bullet';
+
+type SymbolType = 'bullet' | 'numeric' | 'colored-numeric';
+
+type Props = {
+  symbol: SymbolType | React.ReactElement;
+  index: number;
```

Comment:
Is this required when symbol is not numeric?

Reply:
yes...I am always passing it down